### PR TITLE
Create game-streaming vertical page

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Create game-streaming vertical page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8043/gaming
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare against [Copy-doc](https://docs.google.com/document/d/1c9WcMRx1OlQxUntC6YF5oq_wjmwA5vI-RDZC926EYmo/edit#)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/4009

### CODE REVIEW

File: templates/telco/index.html [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+                        <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android app streaming from the cloud.</h4>
+                    </div>
+                </div>
+            </div>
+
+            <div class="p-strip--light">
+                <div class="row u-vertically-center">
+                    <div class="col-7">
+                        <h3>Android in containers</h3>
+                        <p>Unlike Android emulators that make use of virtual machines, Anbox Cloud leverages system containers through LXD, the next generation system container manager. It offers a user experience similar to virtual machines but using lightweight Linux containers instead manageable via a REST API. It also exposes GPUs through a passthrough. You will therefore take advantage of any hardware accelerators to deliver rich mobile experiences.</p>
```
Comment (anthonydillon) [inline templates/telco/index.html:182]: Missing a link to LXD.

File: templates/telco/index.html [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                        }}
+                    </div>
+                </div>
+                <div class="u-fixed-width">
+                    <hr class="p-separator">
+                </div>
+                <div class="row u-vertically-center">
+                    <div class="col-7">
+                        <h3>Software orchestration</h3>
+                        <p>Streaming Android apps to thousands of mobile and desktop clients simultaneously requires orchestration and monitoring of several software components. This involves streaming servers, image repositories, applications, a dashboard application for management and more. Thanks to Juju and MAAS, you can deploy your stack automatically anywhere. </p>
```
Comment (anthonydillon) [inline templates/telco/index.html:202]: Missing links here

File: templates/telco/index.html [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+                        width="289",
+                        height="205",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+                </div>
+                <div class="col-5">
+                    <h3>Invent future mobile services</h3>
+                    <p>Get in touch with Canonical to explore avenues to deliver <br>new value-added services to millions via cloud streaming. </p>
```
Comment (anthonydillon) [inline templates/telco/index.html:234]: Should'nt need a br here. Let the grid wrap the content.

File: templates/telco/index.html [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+                    <button class="p-button--positive">Get in touch</button>
```
Comment (anthonydillon) [inline templates/telco/index.html:235]: Please use this for now: `<a href="/#request-form" class="p-button--positive">Get in touch</a>`

File: templates/gaming/index.html [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+                </p>
+                <p>
+                    Video games were traditionally enjoyed on sophisticated game consoles or expensive desktop PCs. Increasingly, gaming experiences are becoming mobile, spontaneous and casual. This change is driven by the proliferation of smartphones, broadband internet access and the cloud.
+                </p>
+                <p>
+                    With the cloud providing a high-performance and scalable computing backend for games, video streaming is emerging as an attractive distribution channel. Game streaming from the cloud does away with expensive devices. Gamers can play games hosted in the cloud on any smartphone, laptop or tablet with an identical experience as more expensive consoles.
+                </p>
+                <p>
+                    <a href="">
+                        Learn about value-added telco services with Anbox Cloud >
```
Comment (anthonydillon) [inline templates/gaming/index.html:50]: Same here with removing this link

File: templates/gaming/index.html [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+    <div class="row">
+        <div class="col-6">
+            <h2>What’s under the hood</h2>
+            <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android game streaming from the cloud.</h4>
+        </div>
+    </div>
+    <div class="row u-vertically-center">
+        <div class="col-7">
+            <h3>Android in containers</h3>
+            <p>Unlike Android emulators that make use of virtual machines, Anbox Cloud leverages system containers through <a href="https://linuxcontainers.org/">LXD</a>, the next generation system container manager. It offers a user experience similar to virtual machines but using lightweight Linux containers instead manageable via a REST API. It also exposes GPUs through a passthrough. You will therefore take advantage of any hardware accelerators to deliver rich mobile experiences.</p>
```
Comment (anthonydillon) [inline templates/gaming/index.html:151]: Missing `p-link--external` on the LXD link

File: templates/gaming/index.html [hunk 7]
```diff
@@ -0,0 +1,208 @@
...
+</section>
+
+<section class="p-strip">
+    <div class="row">
+        <div class="col-6">
+            <h2>A scalable infrastructure for <br /> game streaming</h2>
+        </div>
+    </div>
+    <div class="row u-equal-height">
+        <div class="col-6 p-card">
```
Comment (anthonydillon) [inline templates/gaming/index.html:116]: Please use to card classes

File: templates/gaming/index.html [hunk 8]
```diff
@@ -0,0 +1,208 @@
...
+            <li class="p-list__item is-ticked">Commercial-grade support for reliability</li>
+        </ul>
+        <p>Anbox Cloud is engineered to address these challenges. The commercially supported platform builds on open source software to empower innovators to stream Android games at scale in the cloud.</p>
+    </div>
+</section>
+
+<section class="p-strip">
+    <div class="row">
+        <div class="col-6">
+            <h2>A scalable infrastructure for <br /> game streaming</h2>
```
Comment (anthonydillon) [inline templates/gaming/index.html:112]: Let the grid wrap the text

File: templates/gaming/index.html [hunk 9]
```diff
@@ -0,0 +1,208 @@
...
+                height="205",
+                hi_def=True,
+                loading="auto"
+                ) | safe
+            }}
+        </div>
+        <div class="col-5">
+            <h3>Stream Android games</h3>
+            <p>Get in touch with Canonical to explore avenues to <br /> deliver your games to millions via cloud streaming. </p>
+            <button class="p-button--positive">Get in touch</button>
```
Comment (anthonydillon) [inline templates/gaming/index.html:203]: Update the link with the comment below

File: templates/gaming/index.html [hunk 10]
```diff
@@ -0,0 +1,208 @@
...
+                width="289",
+                height="205",
+                hi_def=True,
+                loading="auto"
+                ) | safe
+            }}
+        </div>
+        <div class="col-5">
+            <h3>Stream Android games</h3>
+            <p>Get in touch with Canonical to explore avenues to <br /> deliver your games to millions via cloud streaming. </p>
```
Comment (anthonydillon) [inline templates/gaming/index.html:202]: You dont need br's. Lets the grid wrap the content

**Review by anthonydillon — CHANGES_REQUESTED** [review]
QA:
- [ ] Should there be navigation links to these pages?
- [ ] We use 2 spaces in HTML. Can you convert the new pages to use 2 spaces, please?

File: templates/gaming/index.html [hunk 11]
```diff
@@ -0,0 +1,203 @@
...
+        height="205",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-5">
+      <h3>Stream Android games</h3>
+      <p>Get in touch with Canonical to explore avenues to deliver your games to millions via cloud streaming. </p>
+      <button href="/#request-form"  class="p-button--positive">Get in touch</button>
```
Comment (anthonydillon) [inline templates/gaming/index.html:198]: Sorry it looks like there is an contact us page: https://anbox-cloud.io/contact-us

File: templates/telco/index.html (hunk 4 above)
Comment (anthonydillon) [reply templates/telco/index.html:235]: Same here, can link to https://anbox-cloud.io/contact-us

**Review by anthonydillon — CHANGES_REQUESTED** [review]

**Comment by spencerbygraves** [comment]
@petesfrench can we try a couple of changes please.

Can you update the second section as below. The first paragraph is h4.

image

For the last "Stream Android games" section, can you use h2 for the title, and add `is-bordered` to the strip please.

Thanks

File: templates/gaming/index.html [hunk 12]
```diff
@@ -0,0 +1,203 @@
...
+  <div class="p-strip">
+    <div class="row u-align--center">
+      <div class="col-4 u-hide--small">
+        {{ image (
+        url="https://assets.ubuntu.com/v1/767f8475-Anbox_GameStream_Robot.svg",
+        alt="",
+        width="148",
+        height="190",
+        hi_def=True,
+        loading="auto"
```
Comment (anthonydillon) [inline templates/gaming/index.html:60]: Should be lazy loading for all images below the hero/top strip

File: templates/gaming/index.html [hunk 13]
```diff
@@ -0,0 +1,203 @@
...
+        height="205",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+      }}
+    </div>
+    <div class="col-5">
+      <h3>Stream Android games</h3>
+      <p>Get in touch with Canonical to explore avenues to deliver your games to millions via cloud streaming. </p>
+      <button href="/contact-us"  class="p-button--positive">Get in touch</button>
```
Comment (anthonydillon) [inline templates/gaming/index.html:198]:
```suggestion
      <a href="/contact-us"  class="p-button--positive">Get in touch</a>
```

File: templates/telco/index.html [hunk 14]
```diff
@@ -0,0 +1,242 @@
...
+
+{% block content %}
+
+<div class="wrapper">
+    <div id="main-content" class="inner-wrapper">
+        <section class="p-strip--suru-half-and-half-reversed is-dark">
+            <div class="row u-equal-height u-vertically-center">
+              <div class="col-6">
+                <h1>
+                    The 5G mobile <br>subscriber experience
```
[the remark this answers is not in the document]
Comment (anthonydillon) [reply templates/telco/index.html:17]: This was not addressed

File: templates/telco/index.html [hunk 15]
```diff
@@ -0,0 +1,242 @@
...
+                    </p>
+                </div>
+                <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/d5de891a-Acceleration.svg",
+                        alt="",
+                        width="287",
+                        height="160",
+                        hi_def=True,
+                        loading="auto"
```
Comment (anthonydillon) [inline templates/telco/index.html:63]: Should be lazy

**Review by anthonydillon — CHANGES_REQUESTED** [review]
I think you must have forgotten to commit the telco changes as I don't see the comments addressed. Also a few new requests

**Comment by spencerbygraves** [comment]
Thanks @petesfrench, can you please make the hr in the second section the width of the grid only (above the 3 illustrations).

Thanks.

File: templates/gaming/index.html (hunk 13 above)
Comment (anthonydillon) [reply templates/gaming/index.html:203]: Still need to do this

File: templates/gaming/index.html (hunk 12 above)
Comment (anthonydillon) [reply templates/gaming/index.html:64]: This comment too

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments remaining from the previous review.

File: templates/gaming/index.html [hunk 16]
```diff
@@ -0,0 +1,209 @@
...
+    url="https://assets.ubuntu.com/v1/00caf669-Anbox_GameStream_Game.svg",
+    alt="",
+    width="152",
+    height="195",
+    hi_def=True,
+    loading="auto"
+    ) | safe
+    }}
+  </div>
+  </div>
```
Comment (anthonydillon) [inline templates/gaming/index.html:32]: This seems to have some misaligned indentation. Would you reindent please.

File: templates/gaming/index.html [hunk 17]
```diff
@@ -0,0 +1,209 @@
...
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="p-strip u-no-padding--top">
+    <div class="row u-align--center">
+      <div class="col-4 u-hide--small">
```
Comment (anthonydillon) [inline templates/gaming/index.html:58]: You don't need `u-hide--small` on each column but move it up to the parent `p-strip`

File: templates/telco/index.html [hunk 18]
```diff
@@ -8,235 +8,220 @@
...
-                <div class="row">
-                    <div class="col-6">
-                        <h2>What’s under the hood</h2>
-                        <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android app streaming from the cloud.</h4>
-                    </div>
-                </div>
-            </div>
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="row">
```
Comment (anthonydillon) [inline templates/telco/index.html:66]: You have nested rows here. Can you remove this div

File: templates/telco/index.html [hunk 19]
```diff
@@ -8,235 +8,220 @@
...
+      <div class="col-7">
+        <h2>Inventing value-added services for 5G</h2>
+      </div>
+      <p>
+        5G makes ultra-rich media content, games, AR/VR, and even AI, accessible within telco RANs. With unrestricted computing capabilities available on demand, telcos can create innovative subscriber experiences across various mobile form factors to offer differentiated services to their subscribers.
+      </p>
+      <p>
+        What does it take to realise this vision? Considering the scale, this first and foremost requires servers in the cloud with an outstanding cost to performance ratio, equipped with application accelerators like GPUs, FGPAs and ASICs. Furthermore, to minimise latency, mobile workloads should be deployed at the edge, closer to mobile subscribers. Finally, instant elasticity and scalability are required to accommodate subscriber mobility patterns.
+      </p>
```
Comment (anthonydillon) [inline templates/telco/index.html:75]: These elements are outside of a column. I believe they should be contained in the `col-7` above.

File: templates/telco/index.html [hunk 20]
```diff
@@ -8,235 +8,220 @@
...
+          <li class="p-list__item is-ticked">Multi-cloud software automation</li>
+        </ul>
+        <p>
+          Anbox Cloud is engineered to address these challenges. The commercially supported platform builds on open source software to empower innovators to leverage Android at scale in the cloud.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="row u-align--center">
+    <div class="col-4 u-hide--small">
```
Comment (anthonydillon) [inline templates/telco/index.html:95]: Move this `u-hide--small` to the row as we want the entire row hidden on small screens.

File: templates/telco/index.html [hunk 21]
```diff
@@ -8,235 +8,220 @@
...
+        url="https://assets.ubuntu.com/v1/c4b290c8-Contact+us.svg",
+        alt="",
+        width="289",
+        height="205",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-5">
```
Comment (anthonydillon) [inline templates/telco/index.html:219]: This could be a 6 col

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Some requests in the code.

**Comment by spencerbygraves** [comment]
@petesfrench I don't see the change I requested, has the demo been updated? Thanks.

**Comment by petesfrench** [comment]
> @petesfrench I don't see the change I requested, has the demo been updated? Thanks.

Hi Spencer, sorry I think you should be able to see it now.

**Review by anthonydillon — APPROVED** [review]
LGTM :+1: thanks for the changes.

**Pull request merged**